WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -206,11 +206,11 @@ private async Task OnAdjustFeeAsync()
 		var feeRateDialogResult = await NavigateDialogAsync(new SendFeeViewModel(_wallet, _info, false));
 
 		if (feeRateDialogResult.Kind == DialogResultKind.Normal && feeRateDialogResult.Result is { } newFeeRate &&
-		    newFeeRate != _info.FeeRate)
+			newFeeRate != _info.FeeRate)
 		{
 			_info.FeeRate = feeRateDialogResult.Result;
 
-			await BuildAndUpdateAsync(BuildTransactionReason.FeeChanged);
+			_ = BuildAndUpdateAsync(BuildTransactionReason.FeeChanged);
```

Comment:
@soosr I'm abandoning the task here because the suggestions UI has a wait indicator on it, so we can safely fire and forget here. 

Otherwise I was initially planning to add a busy indicator to the entire dialog, but that seems overkill when there is already a smaller one.

Reply:
> @soosr I'm abandoning the task here because the suggestions UI has a wait indicator on it, so we can safely fire and forget here.

That's fine, cACK.

My point is we can have some problems... 

1. You fire and forget only place now... This is not ok because the same issue can happen elsewhere too. 
   - So my first suggestion is to not fire and forget the `BuildAndUpdateAsync` call only when the user changes the fee (And it also affects the building task too which is super bad)
   - `PrivacySuggestions.BuildPrivacySuggestionsAsync()` should be fired and forgotten inside of `BuildAndUpdateAsync` so it would be forgotten no matter if only the fee changes or the selected pockets or anything else.
2. The last 3 times when someone reported a crash on the transaction preview screen... it was all always because of the `PrivacySuggestions.BuildPrivacySuggestionsAsync` and it was awaited so crash happened and we were able to fix it. Would we still be aware of the bugs if we don't await? 
   - `AbandonedTasks` is a class built by Wasabi devs, especially for these cases. It runs the task, awaits it, and logs the error, all in the background. That is why I thought we could use it.

Reply:
Ok, `AbandonedTasks` cannot be used, it only logs in Debug.